tensorflow/lite/kernels/internal/reference/requantize.h

```diff
@@ -45,6 +45,7 @@ inline void Requantize(const input_type* input_data, int32_t size,
       for (int i = 0; i < size; ++i) {
         output_data[i] = input_data[i] ^ 0x80;
       }
+      return;
```

Comment:
this isn't really doing anything. I'd say remove.

Reply:
actually, I take that back. This return is indeed changing the behavior.

The change looks very reasonable to me but let's move this to its own pull request too. Feel free to tag me on the PRs and I can review them quickly.